sdk/sdktypes/symbol.go

```diff
@@ -15,7 +15,10 @@ var InvalidSymbol Symbol
 
 type symbolTraits struct{}
 
-var symbolRE = regexp.MustCompile(`^[a-zA-Z_][\w]*$`)
+var (
```

Comment:
Why the change? IMO now the linter won't catch invalid regexp.

Reply:
i need to reuse the pattern in mcp/tool.go.